src/main/webapp/script.js

```diff
@@ -29,30 +29,60 @@ function loadTasks() {
 }
 
 function renderTasks(tasks) {
-  // TODO: check the size of tasks and show a warning if zero
-  $("#task-list").empty();
-  tasks.forEach(renderSingleTask);
+  const taskList = $("#task-list");
+  taskList.empty();
+  if (tasks.length > 0) {
+    $("#schedule-button").prop("disabled", false);
+    tasks.forEach(renderSingleTask);
+  } else {
+    $("#schedule-button").prop("disabled", true);
+    taskList.html(
```

Comment:
Such HTML injection is not the best practice, mainly because now you have some HTML both in index.html and script file, and some changes (e.g. adding a new MD classes) would require changes in both places.
It would be better to have the message "There are no tasks to schedule" already in the initial DOM, but hidden. In the script you'd just toggle visibility (`.show()`, `.hide()` or `.toggle(condition)`.